app.py

```diff
@@ -15,6 +19,26 @@
 socketio = SocketIO(app)
 socketio.init_app(app, cors_allowed_origins="*")
 
+DOTENV_PATH = join(dirname(__file__), "sql.env")
+load_dotenv(DOTENV_PATH)
+
+# Configure psql database
+DATABASE_URI = os.environ["DATABASE_URL"]
+app.config["SQLALCHEMY_DATABASE_URI"] = DATABASE_URI
+DB = flask_sqlalchemy.SQLAlchemy(app)
+DB.init_app(app)
+DB.app = app
+
+# Create a table
+try:
+    DB.session.execute(
+        "CREATE TABLE users (email VARCHAR(255) PRIMARY KEY,username VARCHAR(255) NOT NULL);"
```

Comment:
I think it's better if we let SQLAlchemy handle the queries and we don't write our own.

If this is related to that issue where SQLAlchemy doesn't make the necessary tables automatically (when it doesn't exist), I know of a workaround we can use.